### PR TITLE
Added `%config(noreplace)` entries for config files

### PR DESCRIPTION
**Pull request by shtripat**
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@r0h4n @nthomas-redhat @TimothyAsir plz review

**Comment by TimothyAsirJeyasing** [comment]
LGTM, However can't we merge all the email related configuration into single yml file. Any reason why we have many config files for email.

File: tendrl-notifier.spec [hunk 1]
```diff
@@ -66,6 +66,10 @@ py.test -v tendrl/notifier/tests || :
 %{_sysconfdir}/tendrl/notifier/email_auth.conf.yaml.sample
```
Comment (TimothyAsirJeyasing) [inline tendrl-notifier.spec]: entries for email_auth.conf.yaml.sample and snmp.conf.yaml files are repeated.
Hope we can remove ln 66 and 67.

Comment (shtripat) [reply tendrl-notifier.spec]: Will do that and push

**Comment by r0h4n** [comment]
@TimothyAsir  review this and close asap pleasae

**Comment by r0h4n** [comment]
@TimothyAsir  any updates on this?

**Comment by r0h4n** [comment]
@TimothyAsir @shtripat  finish this PR soon

**Pull request merged**